docs/faq.rst

```diff
@@ -162,50 +162,44 @@ For example we have an application with two endpoints:
    2. ``/logout_user`` that when invoked needs to close all open
       websockets for that user.
 
-Keep in mind that you can only ``.close()`` a websocket from inside
-the handler task, and since the handler task is busy reading from the
-websocket, it can't react to other events.
-
-One simple solution is keeping a shared registry of websocket handler
-tasks for a user in the :class:`aiohttp.web.Application` instance and
-``cancel()`` them in ``/logout_user`` handler::
+One simple solution is keeping a shared registry of websocket responses
+for a user in the :class:`aiohttp.web.Application` instance and
+call :meth:`aiohttp.web.WebSocketResponse.close` on all of them in ``/logout_user`` handler::
 
     async def echo_handler(request):
 
         ws = web.WebSocketResponse()
         user_id = authenticate_user(request)
         await ws.prepare(request)
-        request.app['websockets'][user_id].add(asyncio.Task.current_task())
-
+        request.app['websockets'][user_id][id(ws)] = ws
```

Comment:
`id(ws)` is not reliable: after garbage collection there is a chance to create new `web.WebSocketResponse()` at the same memory address.